documentation/general/program-scale-testing-requirements/hero-implementation-details.md

```diff
@@ -0,0 +1,247 @@
+
+###### appliance
+
+An enclosure/server hosting multiple DPUs. Combined with a switch can provide a deployment with of the shelf components for an out of server solution.
+
+###### smart switch
+
+Fully integrated solution of a switching ASIC with multiple DPU ASICs. 
+
+
+#### Test Tools packet generator (keysight version)
+
+One solution to test the smart switch is makeing use of Keysight(Ixia) packet generator for TCP traffic we use CloudStorm & IxLoad and for UDP traffic we use Novus & IxNetwork and it is all mixed in by the UHD Connect.
+
+https://github.com/sonic-net/sonic-mgmt/blob/master/docs/testbed/README.testbed.Keysight.md 
+https://www.keysight.com/us/en/products/network-test/network-test-hardware/cloudstorm-100ge-2-port.html
+https://www.keysight.com/us/en/products/network-test/network-test-hardware/novus-qsfp28-1005025ge.html
+https://www.keysight.com/us/en/product/944-1188/uhd400t.html
+https://www.keysight.com/us/en/products/network-test/network-test-hardware/xgs12-chassis-platform.html
+
+Amount of hardware needed varies based on the device performance. Curent DASH requirment specifies 24M CPS as minimum requirment but each vendor wants to showcase how much more they can do so based on that plus adding a 10%-20% for the headroom we can calculate the amount of hardware needed. 
+
+### Puting everything togather
+
+#### Cables
+
+Most cost efective is to use DAC cables, but if equipemnt is in diferent racks optical fibers will be required. We also make use of breakout cables 1x400G to 4x100G to connect UHD400C to CloudSTorm and Novus.
+
+#### Layer 1 considerations
+
+- DASH device port speeds are 100G or 200G or 400G, PAM4 or NRZ are UHD400C device port speeds are 100G or 200G or 400G, PAM4 or NRZ so far the 2 should interface with no issues.
+- IEEE defaults autoneg is preferable but at a minimum if AN is disabled please ensure FEC is enabled. With FEC disabled we observed few packet drops in the DACs and that can create a lot of hasle hunting down a lost packet that has nothing to do with DASH performance.  
+
+#### Testbed examples
+
+Few examples bellow with 100G cables, with 400G cables with fanout cables, single DPU or applaince or smartswitch.
+
+testbed1 
+
+![testbed1](./testbed1.svg)
+
+testbed2 
+
+![testbed2](./testbed2.svg)
+
+testbed3 
+
+![testbed3](./testbed3.svg)
+
+
+>**NOTE**: At this point all hardware should be in the lab powered on, accesible via IP and have link up on all the interfaces.
+
+
+### "1 IP" test
+
+Minimum config posible to run traffic through. 
+
+#### Objectives
+
+##### validate the hardware and software. 
+
+It ensures we can program the DPU via private API, SAI or DASH and that we can pass 1 packets end to end from traffic generator through the device under test and back.
+
+##### can also provide best case scenario performance numbers
+
+its a maybe because 1 packets replicated milions of times may not necesarly work best for all hardware implementations.
```

Comment:
This is very ambiguous. Do you mean, we could take the 1P test case but somehow send more packets to simulate worst-case?

Reply:
rephrased "Not always, but occasionally, this test also yields the best case scenario values because the best case scenario is frequently reached at the lowest scale."